docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -3,12 +3,16 @@ slug: getting-started-with-semgrep-app
 append_help_link: true
 title: Getting started with Semgrep App
 description: "Get started with Semgrep App to scan for security vulnerabilities on cloud repositories hosted on GitHub and GitLab."
-hide_title: true
 ---
 
 import MoreHelp from "/src/components/MoreHelp"
 
-# Scanning cloud repositories with Semgrep App
+Semgrep App supports code scanning from:
+
+* a local command-line interface (CLI)
+* a cloud repository, such as GitHub and GitLab

```

Comment:
Just those two, or are there more? I am thinking if it is more accurate to say “a CI/CD environment, such as GitHub Actions, GitLab CI/CD, or other CI provider environments?”

Reply:
Or maybe it should be a cloud-hosted repository? That would be the case if this refers to code (repositories) that are hosted in an online place, like GitHub.com or GitLab SaaS. However, when I read section `Option B: Adding a cloud repository from GitHub or GitLab` that seems to have instructions for the _CI environments_ of those cloud-hosted repositories / services.

Reply:
We should be very specific about this. Cloud repository not a well defined term. We should say "Source Code Management (SCM) systems like GitHub, GitLab, and BitBucket through CI integration"

Reply:
@s-santillan ok, makes sense. Thanks!